ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java

```diff
@@ -77,13 +77,18 @@ public Bundle build() {
 
   public static AdConfig adConfigWithNetworkExtras(Bundle networkExtras) {
     AdConfig adConfig = new AdConfig();
-    adConfig.setMuted(true); // start muted by default.
+    adConfig.setMuted(false); // Unmute fullscreen ads by default
```

Comment:
actually, it's false by default and will be combined with creative settings

Reply:
yes, here can make ads mute logic run to the same result on adapter side.